step.yml

```diff
@@ -1,6 +1,22 @@
 title: Run npm command
-summary: Runs `npm` with the given command and args.
-description: Runs `npm` with the given command and args.
+summary: The Step runs npm with the command and arguments you provide, for example, to install missing packages or run a package's test.
+description: |-
+  You can install missing JS dependencies with this Step if you insert it before any build step and provide the `npm install` command. You can also test certain packages with the `test` command. You can do both in one Workflow, however, this requires one **Run npm command** Step for installation followed by another **Run npm command** Step for testing purposes.
```

Comment:
> and provide the `npm install` command

You must not include the `npm` part in the command, only the `install`. You can just drop the `npm` part.

Also, please break up long lines for code maintainability purposes.